### DISCUSSION TITLE
Our Client is planning to use content sync tool with aem 6.5.0. Is it possible?

### DISCUSSION THREAD

**Discussion by rxv980** [Adobe-Consulting-Services/acs-aem-commons#3476] [Q&A] [answered]
If no, What is the latest version of aem one should be on to use the content sync functionality?

**Comment by YegorKozlov** [comment]
You can run content sync on AEM 6.5.21, this is the version required by the latest release of ACS Commons.

**Reply by rxv980** [reply]
I've been checking compatability matrix for acs commons and it is very helpful. I got few more questions popped up while going through my research. 

1. I see that content sync is introduced in acs commons **6.2.0**.  Is AEM **6.5.0** compatible with this? 
2. Is service-pkg needs to be installed for acs-commons to be active? (I've tried installing acs-commons 6.4.0 on aem 6.5.0 and acs-commons bundle stays in `installed` status.

**Reply by YegorKozlov** [reply, marked as answer]
> 1. I see that content sync is introduced in acs commons **6.2.0**.  Is AEM **6.5.0** compatible with this?

Yes, it is compatible with AEM 6.5

> 2. Is service-pkg needs to be installed for acs-commons to be active? (I've tried installing acs-commons 6.4.0 on aem 6.5.0 and acs-commons bundle stays in `installed` status.

The latest release ACS Commons 6.9.4 requires aem-service-pkg-6.5.21.0.zip
